examples/imitation_learning_openbot/policies.py

```diff
@@ -5,17 +5,17 @@
 import numpy as np
 import tflite_runtime.interpreter as tflite
 
+import spear
 from utils import get_compass_observation, get_relative_target_pose
 
-
 class OpenBotPIDPolicy():
 
     def __init__(self, config): 
 
         self._config = config
-        self._trajectory = np.empty((0, 3), dtype=float)
+        self._trajectory = np.empty((0, 3), dtype=np.float64)
         self._index_waypoint = 0
-        self._position_xy_old = np.zeros(2, dtype=np.float32)
+        self._position_xy_old = np.zeros(2, dtype=np.float64)
```

Comment:
Use "location" instead of "position" for consistency.

Reply:
Updated this